filebeat/docs/modules/iis.asciidoc

```diff
@@ -0,0 +1,81 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-iis]]
+:modulename: iis
+
+== IIS module
+
+The +{modulename}+ module parses access and error logs created by the
+Internet Information Services (IIS) HTTP server.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+This module requires the
+{elasticsearch-plugins}/ingest-user-agent.html[ingest-user-agent] and
+{elasticsearch-plugins}/ingest-geoip.html[ingest-geoip] Elasticsearch plugins.
+
+The IIS module was tested with logs from version 10.
+
+Note: Running IIS on anything other than Windows is not supported.
```

Comment:
Do we really need that mention? We could always still parse IIS formatted log on a Linux machine.

Reply:
Agreed. I've removed this comment.